app/controllers/import_controller.rb

```diff
@@ -15,16 +15,16 @@ def index
 
   def download_guidance
     send_file(
-      "#{Rails.root}/public/Sara%20Alert%20Import%20Format.xlsx",
-      filename: 'Sara%20Alert%20Import%20Format.xlsx',
+      "#{Rails.root}/public/VACCS%20Import%20Format.xlsx",
+      filename: 'VACCS%20Import%20Format.xlsx',
```

Comment:
@ChristineDuong @kevstoy The interesting thing is that Sara Alert is pointing the download link to the version on GitHub from the master branch. I mirrored the same, so the way this is right now, the link will not work until this is merged into master. 

This is the same file attached here
[VACCS Import Format.xlsx](https://github.com/SaraAlertVaccine/SaraAlert/files/5638413/VACCS.Import.Format.xlsx)

Reply:
[VACCS.Import.Format.xlsx](https://github.com/SaraAlertVaccine/SaraAlert/files/5638812/VACCS.Import.Format.xlsx)

Made small, minor changes:
* Unbolded "Last Date of Exposure" in Format Guidance spreadsheet.
* Added light blue squares to gender identity columns to indicate it was part of the enrollment process in Template and Format spreadsheet

Let me know if that makes sense.